[BUG] 'cpp/cpp' not found

**Describe the bug**
When trying to build tests written to use CMakeTest, I get the following error:
```bash
CMake Error at libs/CMakeTest/cmake_test/cmake_test.cmake:9 (include):
  include could not find requested file:

    cpp/cpp
```

**To Reproduce**
Steps to reproduce the behavior:
1. Set up a minimal CMakePPLang project defining a class.
2. Create a CMakeTest test for the class.
3. Get CMakeTest using this file: [get_cmake_test.cmake.txt](https://github.com/CMakePP/CMakeTest/files/11389568/get_cmake_test.cmake.txt) (removing the ".txt" extension). Put it in the `cmake/` directory and include it with the following command in your `tests/CMakeLists.txt` file:
```
include("${PROJECT_SOURCE_DIR}/cmake/get_cmake_test.cmake")
```
4. "Build" the project and tests with: `cmake -H. -Bbuild -DBUILD_TESTING=ON`
5. See error

**Expected behavior**
There should not be an error finding any requested files. This reference to `cpp/cpp` appears to be referencing CMakePPLang, so it needs to be changed to `cmakepp_lang/cmakepp_lang`.

**Additional context**
A full example can be found in my development fork of CMakePPLang ([link](https://github.com/zachcran/CMakePPLang/tree/joss_cleanup)) under `examples/cmakepplang_example`.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-set(_ct_min_cmake_version "3.14" CACHE STRING "" FORCE)
+set(_ct_min_cmake_version "3.14" CACHE INTERNAL "")
 cmake_minimum_required(VERSION "${_ct_min_cmake_version}") #Required for FetchContent_MakeAvailable()
 project(CMakeTest VERSION 1.0.0 LANGUAGES NONE)
 

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-set(_ct_min_cmake_version "3.14")
+set(_ct_min_cmake_version "3.14" CACHE STRING "" FORCE)
 cmake_minimum_required(VERSION "${_ct_min_cmake_version}") #Required for FetchContent_MakeAvailable()
 project(CMakeTest VERSION 1.0.0 LANGUAGES NONE)
 

--- a/cmake/get_cmaize.cmake
+++ b/cmake/get_cmaize.cmake
@@ -21,6 +21,11 @@ function(get_cmaize)
         )
         FetchContent_MakeAvailable(cmaize)
 
+        set(
+            CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}" "${cmake_test_SOURCE_DIR}/cmake"
+            PARENT_SCOPE
+        )
+
         # Restore the previous value
         set(BUILD_TESTING "${build_testing_old}" CACHE BOOL "" FORCE)
     endif()

--- a/cmake_test/cmake_test.cmake
+++ b/cmake_test/cmake_test.cmake
@@ -6,7 +6,7 @@
 
 include_guard()
 
-include("cpp/cpp")
+include(cmakepp_lang/cmakepp_lang)
 
 # Allows us to capture the root directory of the CMakeTest module
 set(_CT_CMAKE_TEST_ROOT ${CMAKE_CURRENT_LIST_DIR})

--- a/cmake_test/execution_unit.cmake
+++ b/cmake_test/execution_unit.cmake
@@ -1,6 +1,6 @@
 include_guard()
 
-include(cpp/cpp)
+include(cmakepp_lang/cmakepp_lang)
 
 #[[[
 #

